app/utils/dictionary/demography.ts

```diff
@@ -4,1139 +4,1423 @@
+    dialingCode : '+599'
   },
   {
-    name : 'Angola',
-    code : 'AO'
+    name        : 'Angola',
+    code        : 'AO',
+    dialingCode : '+244'
   },
   {
-    name : 'Antarctica',
-    code : 'AQ'
+    name        : 'Antarctica',
+    code        : 'AQ',
+    dialingCode : '+672'
   },
   {
-    name : 'Argentina',
-    code : 'AR'
+    name        : 'Argentina',
+    code        : 'AR',
+    dialingCode : '+54'
   },
   {
-    name : 'American Samoa',
-    code : 'AS'
+    name        : 'American Samoa',
+    code        : 'AS',
+    dialingCode : '+1 684'
   },
   {
-    name : 'Austria',
-    code : 'AT'
+    name        : 'Austria',
+    code        : 'AT',
+    dialingCode : '+43'
   },
   {
-    name : 'Australia',
-    code : 'AU'
+    name        : 'Australia',
+    code        : 'AU',
+    dialingCode : '+61'
   },
   {
-    name : 'Aruba',
-    code : 'AW'
+    name        : 'Aruba',
+    code        : 'AW',
+    dialingCode : '+297'
   },
   {
-    name : 'Åland Islands',
-    code : 'AX'
+    name        : 'Åland Islands',
+    code        : 'AX',
+    dialingCode : '+358'
   },
   {
-    name : 'Azerbaijan',
-    code : 'AZ'
+    name        : 'Azerbaijan',
+    code        : 'AZ',
+    dialingCode : '+994'
   },
   {
-    name : 'Bosnia & Herzegovina',
-    code : 'BA'
+    name        : 'Bosnia & Herzegovina',
+    code        : 'BA',
+    dialingCode : '+387'
   },
   {
-    name : 'Barbados',
-    code : 'BB'
+    name        : 'Barbados',
+    code        : 'BB',
+    dialingCode : '+1 246'
   },
   {
-    name : 'Bangladesh',
-    code : 'BD'
+    name        : 'Bangladesh',
+    code        : 'BD',
+    dialingCode : '+880'
   },
   {
-    name : 'Belgium',
-    code : 'BE'
+    name        : 'Belgium',
+    code        : 'BE',
+    dialingCode : '+32'
   },
   {
-    name : 'Burkina Faso',
-    code : 'BF'
+    name        : 'Burkina Faso',
+    code        : 'BF',
+    dialingCode : '+226'
   },
   {
-    name : 'Bulgaria',
-    code : 'BG'
+    name        : 'Bulgaria',
+    code        : 'BG',
+    dialingCode : '+359'
   },
   {
-    name : 'Bahrain',
-    code : 'BH'
+    name        : 'Bahrain',
+    code        : 'BH',
+    dialingCode : '+973'
   },
   {
-    name : 'Burundi',
-    code : 'BI'
+    name        : 'Burundi',
+    code        : 'BI',
+    dialingCode : '+257'
   },
   {
-    name : 'Benin',
-    code : 'BJ'
+    name        : 'Benin',
+    code        : 'BJ',
+    dialingCode : '+229'
   },
   {
-    name : 'Saint Barthélemy',
-    code : 'BL'
+    name        : 'Saint Barthélemy',
+    code        : 'BL',
+    dialingCode : '+590'
   },
   {
-    name : 'Bermuda',
-    code : 'BM'
+    name        : 'Bermuda',
+    code        : 'BM',
+    dialingCode : '+1 441'
   },
   {
-    name : 'Brunei Darussalam',
-    code : 'BN'
+    name        : 'Brunei Darussalam',
+    code        : 'BN',
+    dialingCode : '+673'
   },
-  {
-    name : 'Bolivia',
-    code : 'BO'
+  {       
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5010957)